web/libs/ui/src/index.ts

```diff
@@ -2,5 +2,8 @@ export * from "./lib/checkbox/checkbox";
 export * from "./lib/Toast/Toast";
 export * from "./lib/label/label";
 export * from "./lib/toggle/toggle";
+export * from "./lib/spinner/Spinner";
+export * from "./lib/space";
+export * from "./lib/simple-card";
```

Comment:
We need to follow the same casing pattern here. All lowercase kebab-case or TitleCase only for components. Default of the generator in NX is kebab-case, so maybe renaming the others to match this.

Reply:
i prefer kebab-case too. will fix